CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-19.6.3
+
+* CLI
+  * Added a check for Pod Security Policy (PSP) permissions to `linkerd check`
+  * Changed `linkerd check` to only warn (not error) if PSPs are found, but
+    the caller does not have the `NET_ADMIN` capability
+* Controller
+  * Improved timestamp formatting in logs
+  * Default the mutating and validating webhook configurations `sideEffects` 
+    property to `None` to indicate that the webhooks have no side effects on
+    external resources
+* Proxy
+  * Added the `NET_RAW` capability to the proxy-init container to prevent it
+    from failing when the primary container has certain security contexts
```

Comment:
I think a bit more specifics here will let users know whether this is relevant to them or not. So, _when the primary container has a `drop: ALL` capability security context_